### DISCUSSION TITLE
UX in Copilot for Visual Studio

### DISCUSSION THREAD

**Discussion by velddev** [community/community#19513] [Copilot Conversations]
Is it by any chance possible to remove the toolbox that comes attached with the copilot tooling? It's really frustrating that I can't read the code that's being recommended

Examples:
![image](https://user-images.githubusercontent.com/13852840/175833108-e0ae83c6-4df3-4649-8ec4-be53802eb57a.png)
![image](https://user-images.githubusercontent.com/13852840/175833145-86c8ae3b-99be-4e93-80f9-43a1594eb67a.png)

**Comment by D7EAD** [comment]
Hi!

I looked everywhere in Visual Studio for an extension setting to disable these tooltips, but I can't seem to find one! As a matter of fact, this appears to be all that is available (as far as I can tell):

![image](https://user-images.githubusercontent.com/42398281/175838158-d7d227c9-b090-469b-b177-58928a74b877.png)

I am unsure if this is the intended behavior of the VS Copilot extension, as in another editors/IDEs such as VSCode, these tooltips appear only once--not every time a suggestion appears!
